spec/rubocop/cop/rspec/rails/minitest_assertions_spec.rb

```diff
@@ -1,73 +1,146 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::RSpec::Rails::MinitestAssertions do
-  it 'registers an offense when using `assert_equal`' do
-    expect_offense(<<~RUBY)
-      assert_equal(a, b)
-      ^^^^^^^^^^^^^^^^^^ Use `expect(b).to eq(a)`.
-    RUBY
-
-    expect_correction(<<~RUBY)
-      expect(b).to eq(a)
-    RUBY
-  end
+  context 'with equal assertions' do
+    it 'registers an offense when using `assert_equal`' do
+      expect_offense(<<~RUBY)
+        assert_equal(a, b)
```

Comment:
It was a miss in the original implementation, it’s confusing that `a` is “expected, and `b` is “actual”.
Up to you if you want to fix this along the way or leave out.

Reply:
Good point - I'll leave it for now since this diff is already pretty big, but happy to do a follow up addressing it